### PR TITLE
Abbreviation plugin

### PR DESCRIPTION
**Pull request by gorzelinski**
This PR updates the "Creating an advanced plugin" tutorial files using new installation methods. They are a supplement for the [guide](https://ckeditor.com/docs/ckeditor5/41.2.1/tutorials/abbreviation-plugin/abbreviation-plugin-level-1.html).

### CODE REVIEW

**Review by illia-stv — CHANGES_REQUESTED** [review]

File: abbreviation-plugin/part-1/package.json [hunk 1]
```diff
@@ -1,40 +1,21 @@
...
   },
-  "name": "part-1",
-  "version": "1.0.0",
-  "description": "Final code for the abbreviation plugin tutorial part 1 (starter files for part 2).",
-  "main": "app.js",
-  "repository": {
-    "type": "git",
-    "url": "git+ssh://git@github.com/ckeditor/ckeditor5-tutorials-examples.git"
+  "dependencies": {
+    "ckeditor5": "^41.3.0-alpha.4"
```
Comment (illia-stv) [inline abbreviation-plugin/part-1/package.json:17]: Leave comment to remember we need replace version with proper one. The same with others.

**Review by illia-stv — APPROVED** [review]

File: abbreviation-plugin/part-2/public/vite.svg
Comment (illia-stv) [inline abbreviation-plugin/part-2/public/vite.svg:1]: Please remove public directories, the same in other parts.

File: abbreviation-plugin/part-1/javascript.svg
Comment (illia-stv) [inline abbreviation-plugin/part-1/javascript.svg:1]: Also remove all svg's generated by vite's CLI like this one.

**Review by Witoso — APPROVED** [review]
I just scanned but looks good 👍

**Review by godai78 — APPROVED** [review]
LGTM

**Pull request merged**